railties/test/isolation/abstract_unit.rb

```diff
@@ -539,6 +539,9 @@ def self.sh(cmd)
 
   assets_path = "#{RAILS_FRAMEWORK_ROOT}/railties/test/isolation/assets"
   unless Dir.exist?("#{assets_path}/node_modules")
+    contents = File.read("#{assets_path}/package.json")
+    contents.sub!(/^RAILS_FRAMEWORK_ROOT/, RAILS_FRAMEWORK_ROOT)
```

Comment:
I think this needs to be changed. `RAILS_FRAMEWORK_ROOT` isn't at the start of the string, so the `^` has to go. There's also more than one occurrence of `RAILS_FRAMEWORK_ROOT` so it needs to be `gsub!` instead of `sub!`

```suggestion
    contents.gsub!(/RAILS_FRAMEWORK_ROOT/, RAILS_FRAMEWORK_ROOT)
```